src/js/msp/MSPHelper.js

```diff
@@ -382,6 +382,19 @@ MspHelper.prototype.process_data = function(dataHandler) {
                     GPS_CONFIG.auto_baud = data.readU8();
                 }
                 break;
+            case MSPCodes.MSP_GPS_RESCUE:
+                if (semver.gte(CONFIG.apiVersion, "1.41.0")) {
```

Comment:
We don't really need the version check here - any firmware that is before 1.41 should never be sent this command, and if it is, it will respond with 'invalid command'.

Reply:
The way the commands are concatenated in the failsafe tab does not differentiate between versions. I will check if it works when invalid command is received.